docs/experiments/extract-mode.md

```diff
@@ -14,7 +14,7 @@ Extract mode enables you to run existing rules on subsections of files where the
 
 Without extract mode, writing rules to validate template, Markdown or configuration files which contain code in another language can be burdensome and require significant rule duplication.
 
-Let's take the following Bash rule as an example (a simplified version of [our `curl-eval` rule](https://github.com/returntocorp/semgrep-rules/blob/release/bash/curl/security/curl-eval.yaml)):
+Let's take the following Bash rule as an example (a simplified version of [`curl-eval` rule](https://github.com/returntocorp/semgrep-rules/blob/release/bash/curl/security/curl-eval.yaml)):
```

Comment:
The general form of " rule" as a way to refer to a specific rule feels very awkward to me. Do we have a standard convention here? If we'd like to avoid possessive determiners, how about "curl-eval, from the Semgrep Registry"?

Reply:
That sounds good, what about: "`curl-eval` rule from the Semgrep Registry!"?

```suggestion
Let's take the following Bash rule as an example (a simplified version of [`curl-eval`](https://github.com/returntocorp/semgrep-rules/blob/release/bash/curl/security/curl-eval.yaml) rule from the Semgrep Registry):
```

Reply:
I am not sure if we had any convention here before, but I really like your second suggestion.

```suggestion
Let's take the following Bash rule as an example (a simplified version of the [`curl-eval`](https://github.com/returntocorp/semgrep-rules/blob/release/bash/curl/security/curl-eval.yaml) rule form the Semgrep Registry):
```